modules/icgc-argo-workflows/prep/metrics/main.nf

```diff
@@ -19,18 +19,39 @@ process PREP_METRICS {
     when:
     task.ext.when == null || task.ext.when
 
-    script: 
+    script:
+    def workflow_name = workflow.Manifest.name
+    
     
-
     """
-    main.py \\
-        -m $multiqc \\
-        -s $meta.id \\
-        -q $qc_files
+    case '$workflow_name' in
+    'Pre Alignment QC')
+        echo $workflow_name detected;
+        prealn.py \\
+            -m $multiqc \\
+            -s $meta.sample \\
+            -q $qc_files
+        ;;
+    'DNA Alignment QC')
```

Comment:
addressing Linda's comment. changed `DNA Alignment` to `DNA Alignment QC`

Reply:
Can you add a separate case for `DNA Alignment QC`. A qc file is generated during `DNA alignment` as well